src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -31,6 +30,8 @@ public class PackageArchiveReader : PackageReaderBase
         /// </summary>
         protected ZipArchive Zip => _zipArchive;
 
+        protected Stream ZipStream { get; set; }
```

Comment:
Remove the set unless it is used, it looks like it is only set by the constructor

Reply:
Add a comment on why this is needed

Reply:
Adding a comet. This is set in `SignedPackageArchive` hence the protected set.